index.html

```diff
@@ -23,10 +25,9 @@ <h2>Seasonal Streams</h2>
       <div>Above you will find all different catagories of movies for any day, time, holiday, or whatever your mood is!</div>

```

Comment:
If you put this div and the <h2> above inside another div and give it the following style you can center these on the page like in your design

```
display: flex
justify-content: center
```

Reply:
Would also recommend changing the color of this text or adding a background, and making it larger to make it more readable!